features/publish_data.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Data extractor should listen to the relevant topic and publish extracted data
```

Comment:
I think, component tests changes should be in a separate PR

Reply:
The component test changes is already merged. I think there is an issue with the rebase . Will fix it